packages/components/credentials/ElasticsearchURLUserPassword.credential.ts

```diff
@@ -0,0 +1,35 @@
+import { INodeParams, INodeCredential } from '../src/Interface'
+
+class ElasticSearchURLUserPassword implements INodeCredential {
+    label: string
+    name: string
+    version: number
+    description: string
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'ElasticSearch Custom URL User Password'
+        this.name = 'elasticSearchURLUserPassword'
+        this.version = 1.0
+        this.description = 'Allow to connect to a custom Elastic index'
+        this.inputs = [
+            {
+                label: 'Custom URL',
```

Comment:
I think we can just add this `customURL` to the existing `ElectricsearchUserPassword.credential.ts` as `type: optional` field

Reply:
In that case we need to set the `cloudId` as optional as well, since the user need to set only one of the two parameters and enter in the right statement in the `ElasticSearchBase.ts`.